subprojects/docs/src/samples/codeQuality/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,69 @@
+// tag::use-checkstyle-plugin[]
+// tag::use-codenarc-plugin[]
+// tag::use-findbugs-plugin[]
+// tag::use-jdepend-plugin[]
+// tag::use-pmd-plugin[]
+plugins {
+// end::use-checkstyle-plugin[]
+// end::use-codenarc-plugin[]
+// end::use-findbugs-plugin[]
+// end::use-jdepend-plugin[]
+// end::use-pmd-plugin[]
+    groovy
+// tag::use-checkstyle-plugin[]
+    checkstyle
+// end::use-checkstyle-plugin[]
+// tag::use-codenarc-plugin[]
+    codenarc
+// end::use-codenarc-plugin[]
+// tag::use-findbugs-plugin[]
+    findbugs
+// end::use-findbugs-plugin[]
+// tag::use-jdepend-plugin[]
+    jdepend
+// end::use-jdepend-plugin[]
+// tag::use-pmd-plugin[]
+    pmd
+// tag::use-checkstyle-plugin[]
+// tag::use-codenarc-plugin[]
+// tag::use-findbugs-plugin[]
+// tag::use-jdepend-plugin[]
+}
+// end::use-checkstyle-plugin[]
+// end::use-codenarc-plugin[]
+// end::use-findbugs-plugin[]
+// end::use-jdepend-plugin[]
+// end::use-pmd-plugin[]
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation(localGroovy())
+    testImplementation("junit:junit:4.12")
+}
+
+// tag::customize-checkstyle-report[]
+tasks.withType<Checkstyle>() {
+    reports {
+        xml.isEnabled = false
+        html.isEnabled = true
+        (html as CustomizableHtmlReport).apply {
+            stylesheet = resources.text.fromFile("config/xsl/checkstyle-custom.xsl")
+        }
+    }
+}
+// end::customize-checkstyle-report[]
+
+// tag::customize-findbugs-report[]
+tasks.withType<FindBugs>() {
+    reports {
+        xml.setEnabled(false)
+        html.setEnabled(true)
+        (html as CustomizableHtmlReport).apply {
```

Comment:
:x: now that #6551 has been merged, you can merge `master` into your branch and remove the cast here

Reply:
Merged `master` and updated sample.